docs/docsite/rst/conf.py

```diff
@@ -238,15 +238,15 @@
     'current_version': version,
     'latest_version': (
         'devel' if tags.has('all') else
-        '2.16' if tags.has('core_lang') or tags.has('core') else
-        '9' if tags.has('ansible')
+        '2.17' if tags.has('core_lang') or tags.has('core') else
+        '10' if tags.has('ansible')
```

Comment:
@samccann I didn't do it when I created the `stable-2.17` branch but we'll need to make the same changes to the conf.py there as well, isn't that so?

Reply:
We can't merge the 2.17 change until core-2.17 releases (see checklist - https://github.com/ansible/ansible-documentation/issues/1249#issuecomment-2037569231)

It also has to be made in two separate PRs. Core becomes 2.17 a month or so before Ansible becomes Ansible 10.  (And similarly, the Ansible 10 change can't be merged until Ansible 10 release day).

Changing earlier than the release day for each of these will mess up either the version switcher or the label of the version above it (I forget which off the top of my head).